src/main/java/org/hisp/dhis/tasks/FilterTeiTask.java

```diff
@@ -0,0 +1,67 @@
+package org.hisp.dhis.tasks;
+
+/*
+ * Copyright (c) 2004-2019, University of Oslo
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ * Redistributions of source code must retain the above copyright notice, this
+ * list of conditions and the following disclaimer.
+ *
+ * Redistributions in binary form must reproduce the above copyright notice,
+ * this list of conditions and the following disclaimer in the documentation
+ * and/or other materials provided with the distribution.
+ * Neither the name of the HISP project nor the names of its contributors may
+ * be used to endorse or promote products derived from this software without
+ * specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
+ * ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import static io.restassured.RestAssured.when;
+
+import io.restassured.response.Response;
+
+/**
+ * @author David Katuscak (katuscak.d@gmail.com)
+ */
+public class FilterTeiTask extends DhisAbstractTask
+{
+    private String endpoint = "/api/trackedEntityInstances.json";
+    private String query = "?ou=DiszpKrYNg8&filter=TfdH5KvFmMy:GE:Karoline&fields=*&includeAllAttributes=true";
+
+    public int getWeight()
+    {
+        return 1;
+    }
+
+    @Override
+    public String getName()
+    {
+        return "Get tracked entity instances matching filter " + endpoint + query;
```

Comment:
I think the name might be too long, but I haven't checked how it looks in locust web. Might give enough information with just "Get TEI's matching filter"? If you think it wouldn't' give enough information and it does look fine in locust web the way it is, then it's fine.

Reply:
That's fine. I changed it to the shorter version. Not really a difference in the meaning for me. Please, merge if fine.